src/main/java/seedu/duke/template/LcTemplate.java

```diff
@@ -37,6 +37,11 @@ public Capacitor getCapacitor() {
         return capacitor;
     }
 
+    public double getImpedance(Inductor l, Capacitor c) {
+        double z = angularFrequency * Math.abs(l.getValue() - (1 / c.getValue()));
+        return z;
+    }
```

Comment:
Can also add JavaDoc.
```suggestion
    public double getImpedance() {
        return angularFrequency * Math.abs(inductor.getValue() - (1 / capacitor.getValue()));
    }
```